src/components/FeaturePanel/PublicTransport/PublicTransport.tsx

```diff
@@ -1,73 +1,73 @@
-import React, { useState, useEffect } from 'react';
-import { useRouter } from 'next/router';
+import React, { useEffect } from 'react';
+import _ from 'lodash';
+import { useQuery } from 'react-query';
 import { Typography } from '@mui/material';
 import { LineInformation, requestLines } from './requestRoutes';
-import { PublicTransportWrapper } from './PublicTransportWrapper';
+import { PublicTransportCategory } from './PublicTransportWrapper';
 import { FeatureTags } from '../../../services/types';
-import { LineNumber } from './LineNumber';
 import { DotLoader } from '../../helpers';
+import { sortBy } from './helpers';
+import { useFeatureContext } from '../../utils/FeatureContext';
+import { getGlobalMap } from '../../../services/mapStorage';
 
 interface PublicTransportProps {
   tags: FeatureTags;
 }
 
-const useLoadingState = () => {
-  const [routes, setRoutes] = useState<LineInformation[]>([]);
-  const [error, setError] = useState<string>();
-  const [loading, setLoading] = useState(true);
+const categories = [
+  'tourism',
+  'commuter',
+  'regional',
+  'long_distance',
+  'high_speed',
+  'night',
+  'car',
+  'car_shuttle',
+  'unknown',
+];
 
-  const finishRoutes = (payload) => {
-    setLoading(false);
-    setRoutes(payload);
-  };
+const PublicTransportDisplay = ({ routes }) => {
+  const grouped = _.groupBy(routes, ({ service }) => {
+    const base = service?.split(';')[0];
+    return categories.includes(base) ? base : 'unknown';
+  });
+  const entries = Object.entries(grouped) as [string, LineInformation[]][];
+  const sorted = sortBy(entries, categories, ([category]) => category);
 
-  const startRoutes = () => {
-    setLoading(true);
-    setRoutes([]);
-    setError(undefined);
-  };
-
-  const failRoutes = () => {
-    setError('Could not load routes');
-    setLoading(false);
-  };
-
-  return { routes, error, loading, startRoutes, finishRoutes, failRoutes };
+  return (
+    <>
+      {sorted.map(([category, lines]) => (
+        <PublicTransportCategory
+          category={category}
+          lines={lines}
+          amountOfCategories={entries.length}
```

Comment:
`amount` is used for uncountable things, `count` is more suitable. 

Or in this case why not call it `showHeading: boolean`? It would be clear from here why we are passing it.